cmd/build/v2/build.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	buildv1 "github.com/okteto/okteto/cmd/build/v1"
 	"github.com/okteto/okteto/pkg/analytics"
+	buildCtx "github.com/okteto/okteto/pkg/build"
```

Comment:
I don't think we should be naming this package `buildCtx`. Right now it only has the `buildCtx` but it can grow and have other types like the `buildInfo`

Reply:
31849e63db2647e6db80ddd77dafe83ad0ae5416